database/database.go

```diff
@@ -117,6 +118,14 @@ func New(config *config.Config) (db *Database, err error) {
 		}
 	}
 
+	if err := db.createServiceTable(); err != nil {
```

Comment:
change db version number and migration script. Or create a task, to not forget it.

Reply:
Will create task for that

Reply:
Done. Task had been created.